en/docs/fragments/register-a-spring-boot-app.md

```diff
@@ -1,55 +1,32 @@
-1. Access WSO2 Identity Server Developer Portal .
- 
- 2. Go to **Applications** and click **New Application**.
-  
- 3. Click **Show More**
- 
- 4. Open the  **OIDC Web Application** template.
-  
- 5. Enter a **Name** for the application 
-       
-    This indicates the name of your application.       
-    **Example**: sample-app
+1.	Start WSO2 IS.
+
+2.	Access the Management Console (https://<IS_HOST\>:<PORT\>/carbon) to create a service provider.
+
+3.	Navigate to the **Service Providers** tab listed under the **Identity** section and click **Add**.
+
+4.	Provide the name for the service provider as `sample-app` and click **Register**. 
+
+5.	Now you will be redirected to the **Edit Service Provider** page.
+
+6.	Expand the  **Inbound Authentication Configuration** section and click **Configure** under the **OAuth/OpenID Connect Configuration** section.
+
+7.	Provide the callback URL as `http://localhost:8080/spring-boot-app/login/oauth2/code/wso2`.
    
- 6. Click **Next**
- 
- 7. Enter **Callback URL**. 
-      Add the flowwing two callBack URLs: 
+   !!!tip
+      The **Callback URL** is the exact location in the service provider's application where a authorization code
+      would be sent. This should be always `{baseUrl}/login/oauth2/code/wso2`.
+
+      Example: http://localhost:8080/login/oauth2/code/wso2
+
+      If you want configure oidc logout, then you need to add a post-logout-url as well.
```

Comment:
shall we also include this example: https://github.com/wso2/docs-is/pull/2460/files#diff-68ed9a8b6ad3cdcce8550ce0e7bcbd73c1fee5cece64d643cd2ac39a14201270L31 ?